app/presenters/table_presenter.rb

```diff
@@ -26,7 +32,7 @@ def headings
   attr_reader :table
 
   def valid_input?
-    return unless %i[headings rows].sort == table.keys.sort
+    return unless %i[show headings rows].sort == table.keys.sort
```

Comment:
What is "show" here? is it the new method?

Reply:
Is there a test for `valid_input?` that needs to be updated?

Reply:
I believe this function checks if the keys in the YAML exist